### DISCUSSION TITLE
vjx<esc> behaves differently if you map the equivalent motions to a key

### DISCUSSION THREAD

**Discussion by nik-rev** [helix-editor/helix#12012] [Troubleshooting] [answered]
With an empty `config.toml`, If we have the following file, with cursor at `*`:

```
This is a v*ery small
poem about sunflowers.
Maybe not.
```

Then pressing `vjx<esc>`, which is equivalent to `select_mode, move_visual_line_down, extend_line_below, normal_mode` will have the following result (visual selection indicated by `||`):

```
||This is a very small
poem about sunflowers.||*
Maybe not.
```

With the following `config.toml`:

```toml
[keys.normal]
j = ["select_mode", "move_visual_line_down", "extend_line_below", "normal_mode"]
```

And the same initial file, and pressing `j`, we will have:

```
This is a v*ery small
||poem about sunflowers.||*
Maybe not.
```

Pressing the keys by hand will select two lines, but binding the equivalent actions to a single key (I tested with different keys like `M`) will select just a single line.

helix version: `helix 24.7 (079f5442)`

**Comment by the-mikedavis** [comment, marked as answer]
Once you've entered select mode the `j` key is instead bound to `extend_visual_line_down` instead of `move_visual_line_down`. You should be able to bind `j = ["extend_visual_line_down", "extend_line_below"]` instead.

**Reply by nik-rev** [reply]
Ok, this makes sense. thanks!!
